Added Intros for Rest APIs

codewithdev opened the pull request:
<!--
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to a pull request) and what changes you've made, we can triage your pull request to the best possible team for review.

See our [CONTRIBUTING.md](/main/CONTRIBUTING.md) for information how to contribute.

For changes to content in [site policy](https://github.com/github/docs/tree/main/content/github/site-policy), see the [CONTRIBUTING guide in the site-policy repo](https://github.com/github/site-policy/blob/main/CONTRIBUTING.md).

We cannot accept changes to our translated content right now. See the [contributing.md](/main/CONTRIBUTING.md#earth_asia-translations) for more information.

Thanks again!
-->

### Why:

Closes #6974

<!-- 
- If there's an existing issue for your change, please link to it.
- If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->

### What's being changed:

<!-- Share artifacts of the changes, be they code snippets, GIFs or screenshots; whatever shares the most context. -->

 - Added Intro for "Codes of conduct"
 - Added Intro for "Code scanning"
 - Added Intro for "Emojis"
 - Added Intro for "GitHub Enterprise administration"
 - Added Intro for "Gists"

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

<!-- Description of the writer impact here -->

codewithdev committed:
Update codes-of-conduct.md

codewithdev committed:
Create codes-of-conduct.md

codewithdev committed:
Update code-scanning.md

codewithdev committed:
Update emojis.md

codewithdev committed:
Update enterprise-admin.md

codewithdev committed:
Update gists.md

codewithdev committed:
Update gists.md

codewithdev committed:
Update codes-of-conduct.md

ramyaparimi commented:
@codewithdev Thanks so much for opening a PR! I'll get this triaged for review :zap:

hubwriter committed:
Update content/rest/reference/emojis.md

hubwriter committed:
Merge branch 'main' into codewithdev-docs-1

hubwriter committed:
Modify the intro text

Remove the initial sentence below the intro to avoid repetition and remove reference to logging as this is not available on GHEC.

hubwriter reviewed (approved):
@codewithdev - Many thanks for adding these intros. I've made a couple of little changes, but otherwise your additions are all great. 🎉 🏅 

I'll get this merged now. Thanks again.
